### PR TITLE
Updated instructions to match OGHO standards

### PR DESCRIPTION
**Pull request by athulmurali**
- Updated ReadME, license
- Added Contribution, security info

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -25,12 +29,51 @@ Once you have the data source installed, configure your datasource with your ten
...
-## Development
-
 ### Debugging
 
 If you want to debug golang backend plugin code, follow the steps below:
 * Install [gops](https://github.com/google/gops) to list running go processes on your machine
 * Run `gops` and find processId for `oci-plugin_darwin_amd64` process
 * Copy this processId to the `.vscode/launch.json`
 * In your VSCode from 'Debug' menu call 'Start Debugging'
+
```
Comment (Djelibeybi) [inline Readme.md:39]: You should only have a single blank line.

File: Readme.md [hunk 2]
```diff
@@ -25,12 +29,51 @@ Once you have the data source installed, configure your datasource with your ten
...
+
+
+## Documentation
+
+> Please refer to  <https://docs.oracle.com>.
```
Comment (Djelibeybi) [inline Readme.md:44]: I'm not sure why this is a block quote. Also, that's a very vague link. You should link specifically to your plugin's documentation.

File: Readme.md [hunk 3]
```diff
@@ -25,12 +29,51 @@ Once you have the data source installed, configure your datasource with your ten
...
+
+## Examples
```
Comment (Djelibeybi) [inline Readme.md:46]: If you have no examples, remove the section.

File: Readme.md [hunk 4]
```diff
@@ -25,12 +29,51 @@ Once you have the data source installed, configure your datasource with your ten
...
+
+> N/A
+
+## Help
+
+  Issues and questions about this plugin can be posted [here](https://github.com/oracle/oci-grafana-plugin/issues)
+
+> Inform users on where to get help or how to receive official support from Oracle
```
Comment (Djelibeybi) [inline Readme.md:54]: Remove this block quote.

File: Readme.md [hunk 5]
```diff
@@ -25,12 +29,51 @@ Once you have the data source installed, configure your datasource with your ten
...
+
+> Please refer to  <https://docs.oracle.com>.
+
+## Examples
+
+> N/A
+
+## Help
+
+  Issues and questions about this plugin can be posted [here](https://github.com/oracle/oci-grafana-plugin/issues)
```
Comment (Djelibeybi) [inline Readme.md:52]: Replace `[here]` with `[as an issue in this GitHub repository]` to make it more accessible.

File: Readme.md [hunk 6]
```diff
@@ -25,12 +29,51 @@ Once you have the data source installed, configure your datasource with your ten
...
+Please consult the [security guide](./SECURITY.md) for our responsible security
+vulnerability disclosure process.
+
+## License
+
+Copyright (c) 2021 Oracle and/or its affiliates.
+
+Released under the Universal Permissive License v1.0 as shown at
+<https://oss.oracle.com/licenses/upl/>.
+
```
Comment (Djelibeybi) [inline Readme.md:73]: Please remove all the blank lines at the end of this file, but the file should end with an `LF` character.

**Review by Djelibeybi — CHANGES_REQUESTED** [review]

File: Readme.md [hunk 7]
```diff
@@ -37,22 +37,13 @@ If you want to debug golang backend plugin code, follow the steps below:
...
-
-
 ## Documentation
 
-> Please refer to  <https://docs.oracle.com>.
-
-## Examples
-
-> N/A
+Please refer to  <https://docs.oracle.com>.
```
Comment (Djelibeybi) [inline Readme.md:42]: This doesn't address the comment at all. It's still way too vague. If you don't have specific documentation for this plugin or the service to which it is associated, remove the section completely.

File: Readme.md (hunk 2 above)
Comment (athulmurali) [reply Readme.md]: Done. Pointed to docs folder as  we don't have any documentation in docs.oracle

**Review by Djelibeybi — APPROVED** [review]

**Pull request merged**